### PR TITLE
Camera's view pyramid visualized

### PR DESCRIPTION
**Pull request by dasimonde**
Added a pyramid to visualize the camera's view direction

### CODE REVIEW

File: src/scene/debug.rs [hunk 1]
```diff
@@ -409,6 +409,27 @@ impl SceneDrawingContext {
         });
     }
 
+    pub fn draw_pyramid(
```
Comment (mrDIMAS) [inline src/scene/debug.rs:412]: Could you please write a few lines of documentation for this?

**Review by mrDIMAS — CHANGES_REQUESTED** [review]
Also, please run `cargo clippy` and fix all the issues it highlights. `cargo fmt` is also needed.

**Comment by dasimonde** [comment]
Sorry, I did it now

**Comment by mrDIMAS** [comment]
Clippy still complains:

```shell
error: this expression creates a reference which is immediately dereferenced by the compiler
   --> src\utils\navmesh.rs:860:55
    |
860 |                 let len = destination.metric_distance(&source);
    |                                                       ^^^^^^^ help: change this to: `source`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow
    = note: `-D clippy::needless-borrow` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::needless_borrow)]`

error: this expression creates a reference which is immediately dereferenced by the compiler
   --> src\utils\navmesh.rs:861:45
    |
861 |                 self.position = source.lerp(&destination, self.interpolator.clamp(0.0, 1.0));
    |                                             ^^^^^^^^^^^^ help: change this to: `destination`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow
```

**Pull request merged**